apps/web-giddh/src/app/settings/profile/setting.profile.component.scss

```diff
@@ -37,7 +37,7 @@
 .branch-vector{
     text-align: right;
     position: absolute;
-    bottom: 8px;
+    bottom: -70px;
```

Comment:
is this -70px working fine in responsive view also, Please check, ignore if already checked.. @shalinee-walkover